msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2018-02-19 03:59+UTC\n"
"PO-Revision-Date: 2018-02-25 21:33-0500\n"
"Last-Translator: Kowlin <boxedpp@gmail.com>\n"
"Language-Team: Bulgarian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.com\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Language: bg\n"
"X-Crowdin-File: /core/locales/messages.pot\n"
"Language: bg_BG\n"

#: ../cog_manager.py:309
msgid "Install Path: {}\n\n"
msgstr "Път на инсталацията: {}\n\n"

#: ../cog_manager.py:325
msgid "That path does not exist or does not point to a valid directory."
msgstr ""

#: ../cog_manager.py:334
msgid "Path successfully added."
msgstr "Пътят е добавен успешно."

#: ../cog_manager.py:347
msgid "That is an invalid path number."
msgstr "Това е невалиден номер на пътечката."

#: ../cog_manager.py:351
msgid "Path successfully removed."
msgstr "Пътят е успешно премахнат."

#: ../cog_manager.py:367
msgid "Invalid 'from' index."
msgstr "Невалиден \"от\" индекс."

#: ../cog_manager.py:373
msgid "Invalid 'to' index."
msgstr "Невалиден \"към\" индекс."

#: ../cog_manager.py:377
msgid "Paths reordered."
msgstr "Пътищата са пренаредени."

#: ../cog_manager.py:395
msgid "That path does not exist."
msgstr "Този път не съществува."

#: ../cog_manager.py:399
msgid "The bot will install new cogs to the `{}` directory."
msgstr "Ботът ще инсталира новите COG-ове в `{}` директория."

#: ../core_commands.py:225 ../core_commands.py:266
msgid "No module by that name was found in any cog path."
msgstr "В директорията на никой COG не е намерен модул с такова име."

#: ../core_commands.py:240
msgid "Failed to load package. Check your console or logs for details."
msgstr "Неуспешно зареждане на пакета. Проверете Вашата конзола или регистрационните файлове за подробни данни."

#: ../core_commands.py:244 ../core_commands.py:253 ../core_commands.py:285
#: ../core_commands.py:374 ../core_commands.py:462 ../core_commands.py:476
msgid "Done."
msgstr "Завършено."

#: ../core_commands.py:255
msgid "That extension is not loaded."
msgstr "Това разширение не е заредено."

#: ../core_commands.py:282
msgid "Failed to reload package. Check your console or logs for details."
msgstr "Неуспешно презареждане на пакета. Проверете Вашата конзола или регистрационните файлове за подробни данни."

#: ../core_commands.py:295
msgid "Shutting down... "
msgstr "Изключване... "

#: ../core_commands.py:310
msgid "Restarting..."
msgstr ""

#: ../core_commands.py:347
msgid "The admin role for this guild has been set."
msgstr "Ролята Администратор е настроена за тази гилдия."

#: ../core_commands.py:355
msgid "The mod role for this guild has been set."
msgstr "Ролята Модератор е настроена за тази гилдия."

#: ../core_commands.py:368
msgid "Failed. Remember that you can edit my avatar up to two times a hour. The URL must be a direct link to a JPG / PNG."
msgstr "Неуспешно. Не забравяйте, че можете да редактирате аватар ми до два пъти на час. URL Адресът трябва да бъде директен линк към JPG / PNG."

#: ../core_commands.py:372
msgid "JPG / PNG format only."
msgstr "Поддържа се JPG / PNG формат само."

#: ../core_commands.py:387
msgid "Game set."
msgstr "Играта е зададена."

#: ../core_commands.py:399
msgid "Listening set."
msgstr ""

#: ../core_commands.py:411
msgid "Watching set."
msgstr ""

#: ../core_commands.py:440
msgid "Status changed to %s."
msgstr "Състоянието се променя на %s."

#: ../core_commands.py:471
msgid "Failed to change name. Remember that you can only do it up to 2 times an hour. Use nicknames if you need frequent changes. `{}set nickname`"
msgstr "Не успяхте да промените името. Не забравяйте, че можете да го направите до 2 пъти на час. Използвайте псевдоними, ако ви се налага нужда от чести промени. `{} зададете псевдоним`"

#: ../core_commands.py:486
msgid "I lack the permissions to change my own nickname."
msgstr "Нямам правомощието да променя псевдонима си."

#: ../core_commands.py:500 ../core_commands.py:513
msgid "Prefix set."
msgstr "Префиксът е зададен."

#: ../core_commands.py:509
msgid "Guild prefixes have been reset."
msgstr "Префиксите на гилдията са нулирани."

#: ../core_commands.py:532
msgid "\n"
"Verification token:"
msgstr "\n"
"Код за потвърждение:"

#: ../core_commands.py:535
msgid "Remember:\n"
msgstr "Запомнете:\n"

#: ../core_commands.py:538
msgid "I have printed a one-time token in the console. Copy and paste it here to confirm you are the owner."
msgstr "Показах еднократен код за потвърждение в конзолата. Копирайте го и го поставете тук за да потвърдите, че вие сте собственика."

#: ../core_commands.py:546
msgid "The set owner request has timed out."
msgstr "Заявката за задаване на собственик изтече."

#: ../core_commands.py:552
msgid "You have been set as owner."
msgstr "Вие сте зададен като собственик."

#: ../core_commands.py:554
msgid "Invalid token."
msgstr "Невалиден ключ."

#: ../core_commands.py:566
msgid "Locale has been set."
msgstr "Локализирането е настроено."

#: ../core_commands.py:580
msgid "Done. Sentry logging is now enabled."
msgstr ""

#: ../core_commands.py:583
msgid "Done. Sentry logging is now disabled."
msgstr ""

#: ../core_commands.py:593
msgid "User ID: %s"
msgstr "Идентификационен номер на потребител: %s"

#: ../core_commands.py:596
msgid "through DM"
msgstr "чрез ЛС"

#: ../core_commands.py:598
msgid "from {}"
msgstr "от {}"

#: ../core_commands.py:599
msgid " | Server ID: %s"
msgstr " | Идентификационен номер на сървъра: %s"

#: ../core_commands.py:608
msgid "Use `{}dm {} <text>` to reply to this user"
msgstr "Използвайте `{} dm {} <text>`, за да отговорите на този потребител"

#: ../core_commands.py:616
msgid "Sent by {} {}"
msgstr "Изпратено от {} {}"

#: ../core_commands.py:628
msgid "I cannot send your message, I'm unable to find my owner... *sigh*"
msgstr "Не мога да изпратя Вашето съобщение, аз не съм в състояние да намеря собственика си... *въздишка*"

#: ../core_commands.py:631
msgid "I'm unable to deliver your message. Sorry."
msgstr "Аз не съм в състояние да доставя Вашето съобщение. Съжалявам."

#: ../core_commands.py:633
msgid "Your message has been sent."
msgstr "Вашето съобщение е изпратено."

#: ../core_commands.py:647
msgid "Invalid ID or user not found. You can only send messages to people I share a server with."
msgstr "Невалиден идентификационен номер или потребителят не е намерен. Можете да изпращате съобщения само до хора, с които споделям общ сървър."

#: ../core_commands.py:653
msgid "Owner of %s"
msgstr "Собственик на %s"

#: ../core_commands.py:657
msgid "You can reply to this message with %scontact"
msgstr "Можете да отговорите на това съобщение с %scontact"

#: ../core_commands.py:667
msgid "Sorry, I couldn't deliver your message to %s"
msgstr "За съжаление, аз не можа да доставя Вашето съобщение до %s"

#: ../core_commands.py:670
msgid "Message delivered to %s"
msgstr "Съобщението е доставено до %s"

#: ../core_commands.py:690
msgid "User added to whitelist."
msgstr ""

#: ../core_commands.py:699
msgid "Whitelisted Users:"
msgstr ""

#: ../core_commands.py:719
msgid "User has been removed from whitelist."
msgstr ""

#: ../core_commands.py:721
msgid "User was not in the whitelist."
msgstr ""

#: ../core_commands.py:729
msgid "Whitelist has been cleared."
msgstr ""

#: ../core_commands.py:746
msgid "You cannot blacklist an owner!"
msgstr ""

#: ../core_commands.py:753
msgid "User added to blacklist."
msgstr ""

#: ../core_commands.py:762
msgid "blacklisted Users:"
msgstr ""

#: ../core_commands.py:782
msgid "User has been removed from blacklist."
msgstr ""

#: ../core_commands.py:784
msgid "User was not in the blacklist."
msgstr ""

#: ../core_commands.py:792
msgid "blacklist has been cleared."
msgstr ""

#: ../dev_commands.py:213
msgid "Already running a REPL session in this channel. Exit it with `quit`."
msgstr "Вече се провежда REPL сесия в този канал. Напуснете я с `quit`."

#: ../dev_commands.py:218
msgid "Enter code to execute or evaluate. `exit()` or `quit` to exit."
msgstr "Въведете код за изпълнение или оценка. `exit()` или `quit` за да излезете."

#: ../dev_commands.py:281
msgid "Unexpected error: `{}`"
msgstr "Неочаквана грешка: `{}`"

